wayland-client/src/lib.rs

```diff
@@ -234,6 +234,15 @@ pub trait Proxy: Clone + std::fmt::Debug + Sized {
     /// The version of this object
     fn version(&self) -> u32;
 
+    /// Checks if the Wayland object associated with this proxy is still alive
+    fn is_alive(&self) -> bool {
```

Comment:
By virtue of having weak proxy/resource wrappers, I'm assuming we needed to readd this.

Reply:
Well, that was not strictly needed, but I realized all pieces were already in place to implement this easily so, might as well do it.